main.go

```diff
@@ -3,103 +3,94 @@ package main
 import (
 	"fmt"
 	"os"
-	"os/exec"
 
-	"github.com/drone/drone-go/drone"
-	"github.com/drone/drone-go/plugin"
+	"github.com/Sirupsen/logrus"
+	"github.com/urfave/cli"
 )
 
-var (
-	buildCommit string
-)
+var buildCommit string // build number set at compile-time
 
 func main() {
-	fmt.Printf("Drone Rubygems Plugin built from %s\n", buildCommit)
-
-	workspace := drone.Workspace{}
-	repo := drone.Repo{}
-	build := drone.Build{}
-	vargs := Params{}
-
-	plugin.Param("workspace", &workspace)
-	plugin.Param("repo", &repo)
-	plugin.Param("build", &build)
-	plugin.Param("vargs", &vargs)
-	plugin.MustParse()
-
-	if (len(vargs.Username) == 0 || len(vargs.Password) == 0) && len(vargs.APIKey) == 0 {
-		fmt.Println("Please provide an API key or username/password credentials")
-
-		os.Exit(1)
-		return
+	logrus.Info(fmt.Printf("Drone Rubygems Plugin built from %s", buildCommit))
```

Comment:
I think I'm going to leave this for now.  I like knowing the commit a plugin was built from for debug purposes.